src/toil_scripts/gatk_germline/germline.py

```diff
@@ -1,688 +1,785 @@
-        #   There are 2 tab-separated columns: UUID and URL
-        #
-        #   UUID        This should be a unique identifier for the sample to be processed
-        #   URL         A URL (http://, ftp://, file://, s3://) pointing to the input SAM or BAM file
-        #
-        #   Example below. Lines beginning with # are ignored.
-        #
-        #   UUID_1    file:///path/to/sample.bam
-        #
-        #   Place your samples below, one per line.
-    """[1:])
-
-
-if __name__ == '__main__':
-
     # Define Parser object and add to jobTree
-    parser = argparse.ArgumentParser()
-    subparsers = parser.add_subparsers(dest='command')
+    parser = argparse.ArgumentParser(description=main.__doc__,
+                                     formatter_class=argparse.RawTextHelpFormatter)
     # Generate subparsers
-    subparsers.add_parser('generate-config', help='Generates an editable config in the current working directory.')
-    subparsers.add_parser('generate-manifest', help='Generates an editable manifest in the current working directory.')
-    subparsers.add_parser('generate', help='Generates a config and manifest in the current working directory.')
+    subparsers = parser.add_subparsers(dest='command')
+    subparsers.add_parser('generate-inputs',
+                          help='Generates an editable inputs in the current working directory.')
+    subparsers.add_parser('generate-manifest',
+                          help='Generates an editable manifest in the current working directory.')
+    subparsers.add_parser('generate',
+                          help='Generates a inputs and manifest in the current working directory.')
+
     # Run subparser
-    parser_run = subparsers.add_parser('run', help='Runs the GATK germline pipeline')
-    group = parser_run.add_mutually_exclusive_group(required=True)
-    parser_run.add_argument('--config', default='gatk_germline.config', type=str,
-                            help='Path to the (filled in) config file, generated with "generate-config".')
-    group.add_argument('--manifest', default='gatk-germline-manifest.tsv', type=str,
-                       help='Path to the (filled in) manifest file, generated with "generate-manifest". '
-                            '\nDefault value: "%(default)s".')
-    group.add_argument('--sample', default=None, nargs='2', type=str,
-                       help='Space delimited sample UUID and BAM file in the format: uuid url')
-    parser_run.add_argument('--output-dir', default=None, help='Full path to directory or filename where '
-                                                               'final results will be output')    
-    parser_run.add_argument('-s', '--suffix', default='.bqsr', help='Additional suffix to add to the names of the output files')
+    parser_run = subparsers.add_parser('run',
+                                       help='Runs the GATK germline pipeline')
+    parser_run.add_argument('--config',
```

Comment:
Personal preference, but I don't find this more readable than 2 lines with `help=` on its own line.